### ISSUE TITLE
Broken link

### ISSUE THREAD

**Issue by srazasyed** [MicrosoftDocs/feedback#3174]
The following link at the bottom of [this ](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/sql?view=azure-dotnet) page is broken:

[Azure management libraries for .NET samples for SQL Database](https://docs.microsoft.com/en-us/dotnet/azure/dotnet-sdk-azure-sql-database-samples)

**Comment by ediulia** [comment]
All the links to dibugdiag at this page and other application pool eventIDs are broken:
https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-R2-and-2008/cc735271(v=ws.10)?redirectedfrom=MSDN

**Comment by rootsmusic** [comment]
@srazasyed Your link, itself, is broken.

**Comment by srazasyed** [comment]
@rootsmusic  Yes. But the question was about that link being broken. That broken link titled `Azure management libraries for .NET samples for SQL Database` at the bottom of this [link](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/sql?view=azure-dotnet) was broken. The broken URL of that broken link was : https://docs.microsoft.com/en-us/dotnet/azure/dotnet-sdk-azure-sql-database-samples. It has been fixed by now on the last line of the article: "The [complete list](https://docs.microsoft.com/en-us/samples/browse/?expanded=azure&products=azure-sql-database%2Cazure-sql-managed-instance%2Cazure-sqlserver-vm%2Cazure-sql-virtual-machines%2Csql-server) of Azure SQL Database samples".
